client/lcd/lcd_test.go

```diff
@@ -51,7 +51,7 @@ func TestKeys(t *testing.T) {
 	var jsonStr = []byte(fmt.Sprintf(`{"name":"test_fail", "password":"%s"}`, password))
 	res, body = Request(t, port, "POST", "/keys", jsonStr)
 
-	assert.Equal(t, http.StatusBadRequest, res.StatusCode, "Account creation should require a seed")
+	assert.Equal(t, http.StatusBadRequest, res.StatusCode, "Account creation should require a seed "+body)
```

Comment:
Sorry, had this in for debugging an error. Should I remove it, as its not related to ineffassign. (That being said, I still think its a useful change)

Likewise for the following change in this function.

Reply:
It's fine.